django/core/serializers/python.py

```diff
@@ -63,6 +63,8 @@ def handle_fk_field(self, obj, field):
                 value = None
         else:
             value = getattr(obj, field.get_attname())
```

Comment:
If we will simply return `field.value_to_string(obj)`, there is a problem for some other fields https://dpaste.de/THba, maybe it is something to do with `value_to_string`